articles/tokens/refresh-token.md

```diff
@@ -1,152 +1,173 @@
 ---
-description: A refresh token allows an application to request Auth0 to issue a new id_token directly, without needing to re-authenticate the user.
+title: Refresh Token
+description: What is a Refresh Token and how you can use it.
 ---
 
-# Refresh Tokens
+# Refresh Token
```

Comment:
Let's add a p saying that refresh tokens are only for Authorization Code, PKCE and password/pwd realm grants. As refresh tokens are super sensitive (you can think of keeping a refresh token as keeping the user u&p) they cannot be placed in a browser.